pkg/storage/plan_general_characteristicsStore.go

```diff
@@ -0,0 +1,99 @@
+package storage
+
+import (
+	_ "embed"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/shared/utilitySQL"
+	"github.com/cmsgov/mint-app/pkg/shared/utilityUUID"
+	"github.com/cmsgov/mint-app/pkg/storage/genericmodel"
+)
+
+//go:embed SQL/plan_general_characteristics_create.sql
+var planGeneralCharacteristicsCreateSQL string
+
+//go:embed SQL/plan_general_characteristics_update.sql
+var planGeneralCharacteristicsUpdateSQL string
+
+//go:embed SQL/plan_general_characteristics_get_by_id.sql
+var planGeneralCharacteristicsGetByIDSQL string
+
+//go:embed SQL/plan_general_characteristics_get_by_model_plan_id.sql
+var planGeneralCharacteristicsGetByModelPlanIDSQL string
+
+// PlanGeneralCharacteristicsCreate creates a new plan basics
+func (s *Store) PlanGeneralCharacteristicsCreate(logger *zap.Logger, gc *models.PlanGeneralCharacteristics) (*models.PlanGeneralCharacteristics, error) {
+	gc.ID = utilityUUID.ValueOrNewUUID(gc.ID)
+
+	statement, err := s.db.PrepareNamed(planGeneralCharacteristicsCreateSQL)
+	if err != nil {
+		return nil, genericmodel.HandleModelCreationError(logger, err, gc)
+	}
+
```

Comment:
After merging, the current practice is to set gc.ModifiedBy and gc.ModifiedDts to null in model creation as a guard against these values being set client-side

Reply:
It's not super clear with the code, but the place I've been setting it is right before the statement.Get(...) calls